src/include/mpir_hw_topo.h

```diff
@@ -183,6 +183,11 @@ MPIR_Node_obj MPIR_Node_get_common_non_io_ancestor_obj(const char *dev_name);
  */
 MPIR_Node_obj_osdev_type MPIR_Node_get_osdev_obj_type(MPIR_Node_obj dev_obj);
 
+/*
+ * Return number of objects of type in the system
+ */
+int MPIR_Node_get_obj_type_num(MPIR_Node_obj_type obj_type);
```

Comment:
Instead of `num`, how about `count`? We are not getting a number (which can mean anything), but we are getting a count, right?

Reply:
Yes I used ret for consistency with the rest of the topology layer. I can change this with count no problem